apps/backend/src/datasources/postgres/TechniqueDataSource.ts

```diff
@@ -108,6 +113,27 @@ export default class PostgresTechniqueDataSource
         throw new Error(`Error getting instruments by technique ID: ${error}`);
       });
   }
+  async getTechniqueScientists(
+    techniqueId: number
+  ): Promise<BasicUserDetails[]> {
+    return database
+      .select('*')
+      .from('users as u')
+      .join('technique_has_scientists as ths', {
+        'u.user_id': 'ths.user_id',
+      })
+      .join('institutions as i', { 'u.institution_id': 'i.institution_id' })
+      .where('ths.technique_id', techniqueId)
+      .then(
+        (
+          usersRecord: Array<UserRecord & InstitutionRecord & CountryRecord>
+        ) => {
+          const users = usersRecord.map((user) => createBasicUserObject(user));
```

Comment:
It is totally fine to return `usersRecord.map((user) => createBasicUserObject(user));` directly, without storing it first.